test/sharness/lib/test-lib.sh

```diff
@@ -45,7 +45,9 @@ test "$TEST_EXPENSIVE" = 1 && test_set_prereq EXPENSIVE
 test "$TEST_NO_DOCKER" != 1 && type docker && test_set_prereq DOCKER
 
 TEST_OS=$(uname -s | tr [a-z] [A-Z])
-test expr "$TEST_OS" : "CYGWIN_NT" >/dev/null && test_set_prereq POSIX
+
+# Set a prereq as error messages are often different on Windows/Cygwin
+expr "$TEST_OS" : "CYGWIN_NT" >/dev/null || test_set_prereq STD_ERR_MSG
```

Comment:
> I don't want to look for a FreeBSD system to test on right now, as I don't think many FreeBSD users have asked for help with running IPFS. And should we also find out for OpenBSD, NetBSD and so on?

This is exactly the point I'm trying to make: let's use `LINUX_OR_OSX` because *we don't know at this moment* whether any other platforms give compatible error messages or not. Let's name things and act based on what behaviours we know, not what we suspect.